src/libostree/libostree.sym

```diff
@@ -360,9 +360,14 @@ global:
  *                         NOTE NOTE NOTE
  */
 
+LIBOSTREE_2016.10 {
+global:
+        ostree_repo_regenerate_summary_ext;
+} LIBOSTREE_2016.8;
+
 /* Remove comment when first new symbol is added
-LIBOSTREE_2016.10
+LIBOSTREE_2016.11 {
```

Comment:
It's just a little weirdness in the ostree symbols file plus weirdness in symbol version files in general. This part is in a comment with a fake symbol waiting for the next release. My addition is above in the .10 section that chains to the .8 section. There's a little extra noise in this patch because the commented out .10 section prior to this patch (there weren't any new symbols yet) was missing the trailing `{`.